src/components/Dropdown/PrioritySelector/PrioritySelector.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Close from '@material-design-icons/svg/round/close.svg';
+import KeyboardDoubleArrowUp from '@material-design-icons/svg/round/keyboard_double_arrow_up.svg';
+import KeyboardArrowUp from '@material-design-icons/svg/round/keyboard_arrow_up.svg';
+import KeyboardArrowDown from '@material-design-icons/svg/round/keyboard_arrow_down.svg';
+import { bem } from '../../../utils/bem';
+import { Dropdown } from '../Dropdown';
+import { ListItem } from '../../List';
+import { Text } from '../../Text';
+import styles from './PrioritySelector.scss';
+import { IconButton } from '../../Buttons';
+
+const iconMap = {
+    mandatory: KeyboardDoubleArrowUp,
+    important: KeyboardArrowUp,
+    optional: KeyboardArrowDown,
+    exclude: Close,
+};
+
+export type Priority = 'mandatory' | 'important' | 'optional' | 'exclude';
+
+export type PriorityItem = {
+    priority: Priority;
+    label: string;
+    value?: string;
```

Comment:
It would be nice to use the generic type here as it was in `SelectedItemBadge`, because, for example, in Search we don't use string, we use custom type, so it will lead to unnecessary type errors